ambient.d.ts

```diff
@@ -19,22 +19,27 @@
  - Name Surname <name.surname@mojaloop.io>
 
  * Google
+ - Abhimanyu Kapur <abhi.kapur09@gmail.com>
  - Steven Wijaya <stevenwjy@google.com>
  --------------
  ******/
 
-import { Client } from '~/shared/ml-thirdparty-client'
+type Client = import('~/shared/ml-thirdparty-client').Client
+type Server = import('@hapi/hapi').Server
+type ServerApplicationState = import('@hapi/hapi').ServerApplicationState
 
-declare module '@hapi/hapi' {
-  interface ServerApplicationState {
-    mojaloopClient: Client
-  }
+interface CustomServerApplicationState extends ServerApplicationState {
+  mojaloopClient: Client
 }
 
-declare module '@mojaloop/sdk-standard-components' {
-  interface response {
-    statusCode: number
-    headers: Record<string, string>
-    data: Record<string, unknown>
-  }
+interface StateServer extends Server {
+  app: CustomServerApplicationState
 }
+
```

Comment:
don't keep commented code, drop it

Reply:
I'm not totally sure why it exists - raman implemented it but doesn't seem to serve a purpose anymore
so i want to keep it around in case there's some future use?